app/models/product.rb

```diff
@@ -26,6 +26,10 @@ class Product < ApplicationRecord
   before_validation :set_name_default
   before_validation :set_discount_price
 
+  scope :enabled_products, -> { where enabled: true }
+  scope :in_line_items, -> { joins(:line_items).distinct }
+  scope :title_in_line_items, -> { in_line_items.pluck(:title) }
```

Comment:
this is NOT a scope. 
Scopes are chainable and should return activerecord relation object (AREL)

Add these as class methods

Reply:
done